tasks/mesh_conf.yml

```diff
@@ -46,11 +46,11 @@
       {{ policies | unique }}
 
 - name: Write tunnel policies for each network
-  lineinfile:
-    dest: "/etc/ipsec.d/policies/{{ item.policy }}"
-    line: "{{ lookup('template', 'policy.j2') }}"
+  template:
+    src: 'policy.j2'
+    dest: "/etc/ipsec.d/policies/{{ item }}"
     mode: '0644'
-  loop: "{{ policies }}"
+  loop: [ 'private', 'clear', 'private-or-clear' ]
```

Comment:
why change from `{{ policies }}`?

Reply:
is it to clear out a policy that isn't in use e.g. if you do the `template:` with `clear`, and there are no `clear` values for `policies`, it will write an empty file for `/etc/ipsec.d/policies/clear`?  If so, then yes, this is good

Reply:
Thank you, @richm. I've pushed another commit 1f0b37483e05e06004ab69c8b21836fbed30b4fd. I think retrieving the policy name from the previously set policies variable should be more straightforward. If it passes the CI tests, I'm removing [WIP] and squash the commits.